### DISCUSSION TITLE
added to install-kernel@jupyter-java

### DISCUSSION THREAD

**Discussion by maxandersen** [dflib/jjava#54] [Ideas]
Awesome you made a refreshed ijava :) 

fyi, I finally got around to add jjava option to install-kernel@jupyter-java (from https://github.com/jupyter-java) so you now can just do:

`jbang install-kernel@jupter-java jjava`

and it works in jupyter.

jbang will handle download of java, the dependencies etc.

**Comment by andrus** [comment]
@maxandersen Great! Let me play with it and add this option to our docs

**Reply by maxandersen** [reply]
fyi, I just updated jupyter-java to use jjava as default kernel (https://github.com/jupyter-java)

**Reply by andrus** [reply]
Perfect! While there as still some "dark spots" that we inherited from iJava that we need to inspect, document and maybe improve, but since the M4 I personally didn't have any issues at all. Been very stable and reliable.

**Reply by andrus** [reply]
@maxandersen , BTW BlueSky shows your profile to have an "Invalid Handle" for some reason. So I couldn't tag you when I reposted it there - https://bsky.app/profile/andrus.adamchik.org/post/3ldqramfhj22p

**Reply by maxandersen** [reply]
Weird. It should be maxandersen.xam.dk

**Reply by andrus** [reply]
Aha. Probably a Bluesky bug 
<img width="349" alt="image" src="https://github.com/user-attachments/assets/7f900e9e-7a94-4570-bc0d-5e69ed8355ec" />

**Reply by maxandersen** [reply]
<img width="335" alt="image" src="https://github.com/user-attachments/assets/6a14f9dc-11e1-46b0-9437-5a78d3318c96" />

one of us is living a lie :)
